setup.py

```diff
@@ -26,12 +26,12 @@ def _resolve_prefix(prefix, type):
     # issue: https://github.com/vcstools/wstool/issues/81
     osx_system_prefix = '/System/Library/Frameworks/Python.framework/Versions'
     if type == 'man':
-        if prefix == '/usr':
+        if prefix == '':
             return '/usr/share'
         if sys.prefix.startswith(osx_system_prefix):
             return '/usr/local/share'
     elif type == 'bash_comp':
-        if prefix == '/usr':
+        if prefix == '':
```

Comment:
It's more complicated than it seems, unfortunately. There's quite a bit of history on this particular topic, part of it stems from differences in OS's and part is that the FHS isn't consistent when installing to `/usr` versus other locations. For example, the etc folder when installing to `/usr` is `/etc`, but when installing to `/usr/local` it is `/usr/local/etc`. That's the origin of these special conditions. You can read some about it in other issues:

https://github.com/vcstools/wstool/pull/82
https://github.com/catkin/catkin_tools/issues/222
https://github.com/catkin/catkin_tools/pull/223

This pr might be more like what we should do here:

https://github.com/catkin/catkin_tools/pull/296